### PR TITLE
Add links to the pointsource files

### PR DESCRIPTION
**Pull request by Bo-Li-LMU**
Here I add the repository link for the point source example.

### CODE REVIEW

**Comment by krenzland** [comment]
Thanks for your contribution!
Should we wait with merging until https://github.com/SeisSol/Examples/pull/1 is finished or should we merge this now and change the link later=

**Comment by Thomas-Ulrich** [comment]
Links should be formatted as follows:

```
`proj.4 <https://github.com/OSGeo/proj.4>`__
```

**Comment by Bo-Li-LMU** [comment]
I think we can add the link now then update later. If someone is not familiar with the SeisSol repository, then it's hard to find and try the example.

File: Documentation/pointsource.rst [hunk 1]
```diff
@@ -12,7 +12,8 @@ Validation interface established within the 6th Framework Programme
 project.
 
 LOH1 is used as an example here to illustrate the implementation of
-source point for earthquake nucleation in SeisSol. The details of the LOH1 model can also be found at .
+source point for earthquake nucleation in SeisSol. The details of the LOH1 model can also be found at 
+https://github.com/daisy20170101/SeisSol_Cookbook/tree/81dc61fa5e8e7c2583ab086a9a54c482bcb157f5/PointSource.
```
Comment (Thomas-Ulrich) [inline Documentation/pointsource.rst:16]: https://github.com/SeisSol/Examples/tree/master/WP2_LOH1

File: Documentation/pointsource.rst [hunk 2]
```diff
@@ -60,7 +61,7 @@ The point source needs to be turned on in *parameter.par* file.
   FileName=’LOH1_source.dat’
   /
 
-The source input file can be found at . Duration of the source is 4
+The source input file can be found at above link. Duration of the source is 4
```
Comment (Thomas-Ulrich) [inline Documentation/pointsource.rst:64]:
```
can be generated using  `this script <https://github.com/SeisSol/Examples/blob/master/WP2_LOH1/generate_LOH_source_type50.py>`__
```

Comment (Bo-Li-LMU) [reply Documentation/pointsource.rst]: Thanks everyone. Now the link is updated and linked to the master branch Examples.

**Review by Thomas-Ulrich — APPROVED** [review]

**Review by krenzland — CHANGES_REQUESTED** [review]
Please use a new branch for each merge commit - or at least wait, until the changes are merged into master.
This invalidates Thomas' code review.

**Comment by Bo-Li-LMU** [comment]
OK, I'll remove the new updates and and then upload after merging.

**Comment by Thomas-Ulrich** [comment]
@krenzland can you approve the PR?

**Review by krenzland — APPROVED** [review]

**Comment by krenzland** [comment]
Sure, sorry

**Pull request merged**